Different guava versions in dependencies

Version 2.7.0.

Output of maven-enforcer-plugin:
`  +-io.springfox:springfox-swagger2:2.7.0`
`    +-io.springfox:springfox-spi:2.7.0`
`      +-io.springfox:springfox-core:2.7.0`
`        +-com.google.guava:guava:18.0`

`  +-io.springfox:springfox-swagger2:2.7.0`
`    +-io.springfox:springfox-spring-web:2.7.0`
`      +-org.reflections:reflections:0.9.11`
`        +-com.google.guava:guava:20.0`

It breaks for me.  I am seeing the following on a spring framework project during startup when moving from 2.6.1 to 2.7.0.

Caused by: java.lang.NoSuchMethodError: com.google.common.math.IntMath.saturatedAdd(II)I

Intellij shows 2 versions of Guava as described above.  Rather than add the extra dependency I have just moved back to 2.6.1.  I am using tomcat 8.5.16.

--- a/buildSrc/build.gradle
+++ b/buildSrc/build.gradle
@@ -23,7 +23,7 @@ repositories {
 dependencies {
   compile gradleApi()
   compile localGroovy() //Gradle 2.3 -> Groovy 2.3.9
-  compile "com.google.guava:guava:18.0"
+  compile "com.google.guava:guava:20.0"
   testCompile 'org.spockframework:spock-core:1.1-groovy-2.4-rc-4@jar'
   testCompile 'junit:junit:4.12'
 }

